internal/pkg/cli/storage_init.go

```diff
@@ -46,15 +47,88 @@ We'll deploy the Cloudformation for the storage when you run 'svc deploy'.`
 
 var fmtStorageInitCreateConfirm = "Okay, we'll create %s %s named %s linked to your %s service."
 
+var fmtStorageInitDynamoInfoText = `We're going to ask you some questions about your database schema and datatypes.
+You can change any of this information at the end by editing the CloudFormation
+generated at ./ecs-project/%s/addons/%s-%s.yml.`
+
+var (
+	fmtStorageInitDDBKeyPrompt  = "What would you like to name the %s of this %s?"
+	storageInitDDBKeyTypePrompt = "What datatype is this key?"
+	storageInitDDBLSIPrompt     = "Would you like to add a local secondary index to this table?"
+	storageInitDDBLSINamePrompt = "What would you like to name this " + color.Emphasize("LSI") + "?"
+)
+
+var (
+	storageInitDDBPartitionKeyPromptHelp = "The partition key of this table. This key, along with the sort key, will make up the primary key."
+	storageInitDDBSortKeyPromptHelp      = "The sort key of this table. Without a sort key, the partition key " + color.Emphasize("must") + " be unique on the table."
+	storageInitDDBLSISortKeyPromptHelp   = "The sort key of this Local Secondary Index. An LSI can be queried based on the partition key and LSI sort key."
+	storageInitDDBKeyTypeHelp            = "The datatype to store in the key. N is number, S is string, B is binary."
+	storageInitDDBLSIHelp                = "A Local Secondary Index has the same partition key as the table, but a different sort key."
+)
+
+const (
+	StringType = "S"
```

Comment:
can these be private? and maybe to avoid collisions within the cli pkg `ddbStringAttributeType = "S"`